CrisisTextLine/Elmer
# Scrub twilio history

#### What's this PR do?
Scrubs twilio messages from the previous day
#### Where should the reviewer start?
At the top
#### How should this be manually tested?
NPM install 
Add logging to `redactMessages(data.messages)
  .then((redactedMsgs) => {` So you can see the progression of pages and let it run until it ends
#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=12&selectedIssue=AT-47&quickFilter=44
#### Questions:
Is there a better way of summing totalRedactedMsgs? (possibly adding some recursive returns?)
